.changeset/grumpy-coats-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': minor
```

Comment:
patch?

```suggestion
'@primer/react': patch
```

Reply:
@siddharthkp my thinking for picking `minor` was that this change seemed like it was adding functionality where a type was broadended to accept more values. Right now in our versioning doc we have that as a `minor` over at: https://github.com/primer/react/blob/main/contributor-docs/versioning.md#the-type-of-a-prop-is-broadened 

Let me know if this seems more like a bug fix though or if it'd be worth talking about updating the version doc 👍